apps/platform/trpc/routers/authRouter/passwordRouter.ts

```diff
@@ -74,8 +76,9 @@ export const passwordRouter = router({
       return { success: true };
     }),
 
-  signUpWithPassword2FA: publicRateLimitedProcedure.signUpWithPassword
+  signUpWithPassword2FA: publicProcedure
     .unstable_concat(turnstileProcedure)
+    .use(ratelimiter({ limit: 10, namespace: 'signUp.password' }))
```

Comment:
duplicate name @BlankParticle

Reply:
one of the route is not used and is kept for not having issues in the old Nuxt app

Reply:
ah ok